math/geometry/distance_test.go

```diff
@@ -0,0 +1,77 @@
+package geometry
```

Comment:
```suggestion
package geometry_test
```

This would mean you won't be able to call your function directly. You would need to import the `geometry` package like this


```Go
import "github.com/TheAlgorithms/Go/math/geometry
```

and calling things would require you to prefix it like this

```Go
geometry.EuclideanPoint{}
```
etc.